src/.vuepress/config.js

```diff
@@ -1,14 +1,14 @@
 const sidebar = {
   cookbook: [
     {
-      title: 'Cookbook',
+      title: 'Recetario',
       collapsable: false,
       children: ['/cookbook/', '/cookbook/editable-svg-icons']
     }
   ],
   guide: [
     {
-      title: 'Essentials',
+      title: 'Esenciales',
```

Comment:
I think we should use `Conocimientos Esenciales` to keep it consistent with the current docs
```suggestion
      title: 'Conocimientos Esenciales',
```